### ISSUE TITLE
Deprecation warnings for set-state and set-output commands

### ISSUE THREAD

**Issue by jjoynson-iw** [pnpm/action-setup#57]
Seeing the following two warnings when using this action:

```
The `save-state` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by belgattitude** [comment]
Just proposed two PR's about this

- #58  
- #59 

The doc is simple, the action/core update probably needs a second eye.

**Comment by zkochan** [comment]
Isn't the warning printed by the programmatic usage? https://github.com/pnpm/action-setup/blob/537643d491d20c2712d11533497cb47b2d0eb9d5/src/outputs/index.ts#L8-L9

**Comment by belgattitude** [comment]
Seing the PR merged into action/core 1.10.0, seems they've updated the internals of setOutput

https://github.com/actions/toolkit/pull/1178/files

My best guess hypothesis is that there's nothing to do apart updating.

**Comment by zkochan** [comment]
published v2.2.4

**Comment by belgattitude** [comment]
Thanks @zkochan, I've tested it and no warnings are emitted anymore. I think this can be closed
